src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import Map from '../../Map/Map.js';
+
+class OrganizationDetails extends Component {
+
+    render() {
+        return (
+            <div className="scs-0">
+                <div className="scs-0-image">
+                    <Map
+                        marginLeft="20px" 
+                        height="150px" 
+                        width="70%" 
+                        marginTop="10px" 
+                        marginRight="0px"
+                        marginBottom="0px"
+                        address={this.props.account.address}
+                    />
+                </div>
+
+                {!this.props.isEditingOrg ?
+
+                    <div className="scs-0-content">
+                        <h5>Organization Details</h5>
+                        <h6>{this.props.account.name}</h6>
+                        <div className="scs-spacing" />
+                        <h6>Address: {this.props.account.address.street1} {this.props.account.address.street2} {this.props.account.address.city}, {this.props.account.address.state} {this.props.account.address.zipcode}</h6>
```

Comment:
- nit-pick: should probably have a comma between street1 and street2
- there should be a conditional rendering for address.officeNo
```
{this.props.account.address.officeNo &&
    <h6>Office No: {this.props.account.address.officeNo}</h6>
}
```

Reply:
Fixed.